server/routes/api.js

```diff
@@ -0,0 +1,91 @@
+const express = require('express')
+const router = express.Router()
+const mongoose = require('mongoose')
+const Expense = require('../models/Expense')
+let moment = require('moment')
+mongoose.connect("mongodb://localhost/mongoose-exercise",  {useNewUrlParser: true,  useUnifiedTopology: true } )
```

Comment:
There is a connection to MongoDB in the server.js file. no need to connect it again